README.md

```diff
@@ -30,29 +30,35 @@ You will need the following things properly installed on your computer.
 * `ember test`
 * `ember test --server`
 
-### Building
+### Manual Builds
 
-* `ember build` (development)
-* `JOB_WATCHER_URL="wss://bespin-ws.gcb.duke.edu" ember build --environment production` (production) (see `build-release.sh`)
+* `ember build` (Local Development)
+* `JOB_WATCHER_URL="wss://bespin-ws.gcb.duke.edu" ember build --environment production` (production) (see below)
 
-### Deploying
-
-This application is built and bundled with [bespin-api](https://github.com/Duke-GCB/bespin-api) in [https://github.com/Duke-GCB/bespin-webapp-docker](Duke-GCB/bespin-webapp-docker)
-
-When a tag is pushed, [Travis-CI](https://travis-ci.org/Duke-GCB/bespin-ui) will create a production build and upload it as `bespin-ui.tar.gz` to the GitHub release corresponding to the tag, as described in [.travis.yml](.travis.yml). Travis also triggers the Docker Hub build of bespin-webapp-docker.
+#### Release Process
 
```

Comment:
This confused me. Reading this sounds like I should manually do the following steps.
The next section seems to walk this back by saying just run `npm version patch`.
Are these steps just documenting what the `npm version patch` command does?
Could we merge these two sections? - lead with the single command to run and document the details of what it is doing below (if necessary).